themes/powerline/powerline.base.bash

```diff
@@ -41,6 +41,24 @@ function __powerline_user_info_prompt {
   [[ -n "${user_info}" ]] && echo "${user_info}|${color}"
 }
 
+function __powerline_terraform_prompt() {
+  local terraform_workspace=""
+
+  if [ -d .terraform ]; then
+    terraform_workspace="$(command terraform workspace show 2>/dev/null)"
+    [[ -n "${terraform_workspace}" ]] && echo "${TERRAFORM_CHAR}${terraform_workspace}|${TERRAFORM_THEME_PROMPT_COLOR}"
+  fi
+}
+
```

Comment:
Looks like you forgot to delete the older version of the function from the PR?

Reply:
Fixed